packages/editor/src/components/post-title/index.js

```diff
@@ -116,7 +116,9 @@ class PostTitle extends Component {
 								*/
 								/* eslint-disable jsx-a11y/no-autofocus */
 								autoFocus={
-									document.body === document.activeElement &&
+									( document.body ===
+										document.activeElement ||
+										! document.activeElement ) &&
```

Comment:
Aside: I would have liked to avoid the ambiguity here of `===` in possibly evaluating as "equal to the result of `document.activeElement || ! document.activeElement`" (wrong), but apparently Prettier is opinionated about this, and will not allow for the (apparently redundant) extra parentheses around `( document.body === document.activeElement )`.